cmd/app/options/options.go

```diff
@@ -193,6 +194,10 @@ func (t *TLSOptions) addFlags(fs *pflag.FlagSet) {
 		"Certificate duration of serving certificates. Will be renewed after 2/3 of "+
 			"the duration.")
 
+	fs.StringSliceVar(&t.DNSNames,
+		"dns-names", []string{"cert-manager-istio-csr.cert-manager.svc"},
```

Comment:
Just to keep the naming consistent.
```suggestion
		"serving-certificate-dns-names", []string{"cert-manager-istio-csr.cert-manager.svc"},
```